lib/main.js

```diff
@@ -1,3 +1,4 @@
+// foo bar asd as sdfsdf
```

Comment:
There is a JavaScript syntax error in your code; validation cannot continue on this file.